pipeline.py

```diff
@@ -72,31 +90,102 @@ class TextProcessingPipeline:
     """
 
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        articles = self.corpus_manager.get_articles().values()
+        for article in articles:
+            raw_text = article.get_raw_text()
+
+            cleaned_tokens = []
+            single_tagged_tokens = []
+            multiple_tagged_tokens = []
+
+            for token in self._process(raw_text):
+                cleaned_tokens.append(token.get_cleaned())
+                single_tagged_tokens.append(token.get_single_tagged())
+                multiple_tagged_tokens.append(token.get_multiple_tagged())
+
+            article.save_as(' '.join(cleaned_tokens), kind='cleaned')
+            article.save_as(' '.join(single_tagged_tokens), kind='single_tagged')
+            article.save_as(' '.join(multiple_tagged_tokens), kind='multiple_tagged')
```

Comment:
as it is stated in the 5.3 section of pipeline documentation, you are required to pass `ArtifactType` attributes as a `kind` argument to the `save_as() method`, this way code is more secure as it is protected from almost untraceable errors caused by simple typos